### ISSUE TITLE
Hit and run

### ISSUE THREAD

**Issue by mottosso** [pyblish/pyblish-qml#205]
### Description

Telling QML to show, and then immediately exit causes QML to stop responding.
### To reproduce

**Terminal 1**

``` bash
$ python -m pyblish_qml --demo
```

**Terminal 2**

``` bash
$ python -c "import pyblish_qml;pyblish_qml.show()"
```

This command immediately exits, before giving QML a chance to acknowledge the request, causing it to stop listening for future requests.

**Comment by tokejepsen** [comment]
When quitting while processing ```pyblish-qml``` gets the same problem.

**Reproducible**
```python
import subprocess
import time


# server
p_server = subprocess.Popen(["python", "-m", "pyblish_qml"])

script = """import time
import pyblish_qml
import pyblish.api
class LongDelay(pyblish.api.ContextPlugin):

    order = pyblish.api.CollectorOrder

    def process(self, context):
        time.sleep(30)

pyblish.api.register_plugin(LongDelay)


pyblish_qml.install()
pyblish_qml.show()
try:
    while True:
        time.sleep(1)
except (KeyboardInterrupt, SystemExit):
    # Close GUI on terminal session end
    pass
"""

# client 1
p_client = subprocess.Popen(["python", "-c", script])
time.sleep(5)
p_client.kill()

# client 2
p_client = subprocess.Popen(["python", "-c", script])
```

This has been discussed in the forums as well; http://forums.pyblish.com/t/pyblish-standalone-exit-prematurely/267

**Comment by mottosso** [comment]
Solved in 1.0.0
